src/ado2gh/Commands/MigrateRepoCommand.cs

```diff
@@ -41,6 +42,11 @@ public class MigrateRepoCommand : Command
             {
                 IsRequired = true
             };
+            var targetRepoVisibility = new Option<string>("--target-repo-visibility")
```

Comment:
I'm of two minds about the naming of this: on the one hand, I like staying consistent, on the other hand, only `gei` really cares about `source` vs. `target` repo, and we can just call this `--github-repo-visibility` here.

Reply:
I could go either way, @timrogers or @roferg any opinion? when we add `--target-repo-visibility` to the 3 different migrate-repo commands, should we be consistent with `--target-repo-visibility` as the arg name for all 3, or should we do `gh gei migrate-repo --target-repo-visibility` 'gh ado2gh migrate-repo --github-repo-visibility` `gh bbs2gh migrate-repo --github-repo-visibility`

Reply:
We should be consistent and stick with the `target` branding. So I'd use `--target-repo-visibility`.

Reply:
I'm cool with that, but to provide a bit more context, we already differ argument names between the 3 extensions. For example:
`ado2gh migrate-repo --ado-repo --github-repo`
`gei migrate-repo --source-repo --target-repo`
`bbs2gh migrate-repo --bbs-repo --github-repo`

Reply:
So Option 1 would be:
`ado2gh migrate-repo --ado-repo foo --github-repo foo --target-repo-visibility internal`
`gei migrate-repo --source-repo foo --target-repo foo --target-repo-visibility internal`
`bbs2gh migrate-repo --bbs-repo foo --github-repo foo --target-repo-visibility internal`

Option 2 would be:
`ado2gh migrate-repo --ado-repo foo --github-repo foo --github-repo-visibility internal`
`gei migrate-repo --source-repo foo --target-repo foo --target-repo-visibility internal`
`bbs2gh migrate-repo --bbs-repo foo --github-repo foo --github-repo-visibility internal`

Reply:
Did we change to GitHub repo recently? Our docs for ADO2GH use `--target-repo`. My opinion would be to align uniformly across all of the extensions unless there is a unique situation that calls for difference. In this case that would be using `target`.

Reply:
that's an oversight docs are currently incorrect for ado2gh